core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -227,6 +227,11 @@ private boolean isCompactWool() {
         && MAX_ROWS < wmm.getWools().keySet().size() * 2 - 1 + wmm.getWools().values().size();
   }
 
+  private boolean isSuperCompactWool() {
```

Comment:
```suggestion
  private boolean isSuperCompact() {
```
Maybe this shouldn't be restricted to just compacting by wools, if other objectives need to show maybe it's worth removing the in-between-team lines in the scoreboard. 
Obviously the method should change accordingly to add up non-wool objectives, one line for compacted wools, and one line for the space between teams